datasets/cms_medicare/inpatient_charges/pipeline.yaml

```diff
@@ -0,0 +1,702 @@
+            mode: "NULLABLE"
+          - description: "The city in which the provider is physically located"
+            name: "provider_city"
+            type: "STRING"
+            mode: "NULLABLE"
+          - description: "The state in which the provider is physically located"
+            name: "provider_state"
+            type: "STRING"
+            mode: "NULLABLE"
+          - description: "The zip code in which the provider is physically located"
+            name: "provider_zipcode"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - description: "The code and description identifying the MS-DRG. MS-DRGs are a classification system that groups similar clinical conditions (diagnoses) and the procedures furnished by the hospital during the stay"
+            name: "drg_definition"
+            type: "STRING"
+            mode: "REQUIRED"
+          - description: "The Hospital Referral Region (HRR) in which the provider is physically located"
+            name: "hospital_referral_region_description"
+            type: "STRING"
+            mode: "NULLABLE"
+          - description: "The number of discharges billed by the provider for inpatient hospital services"
+            name: "total_discharges"
+            type: "INTEGER"
+            mode: "NULLABLE"
+          - description: "The provider's average charge for services covered by Medicare for all discharges in the MS-DRG. These will vary from hospital to hospital because of differences in hospital charge structures"
+            name: "average_covered_charges"
+            type: "FLOAT"
+            mode: "NULLABLE"
+          - description: "The average total payments to all providers for the MS-DRG including the MSDRG amount, teaching, disproportionate share, capital, and outlier payments for all cases. Also included 5 in average total payments are co-payment and deductible amounts that the patient is responsible for and any additional payments by third parties for coordination of benefits"
+            name: "average_total_payments"
+            type: "FLOAT"
+            mode: "NULLABLE"
+          - description: "The average amount that Medicare pays to the provider for Medicare's share of the MS-DRG. Average Medicare payment amounts include the MS-DRG amount, teaching, disproportionate share, capital, and outlier payments for all cases. Medicare payments DO NOT include beneficiary co-payments and deductible amounts nor any additional payments from third parties for coordination of benefits"
+            name: "average_medicare_payments"
+            type: "FLOAT"
+            mode: "NULLABLE"
+
+  graph_paths:
+    - "inpatient_2011_transform_csv >> load_inpatient_2011_to_bq"
+    - "inpatient_2012_transform_csv >> load_inpatient_2012_to_bq"
+    - "inpatient_2013_transform_csv >> load_inpatient_2013_to_bq"
+    - "inpatient_2014_transform_csv >> load_inpatient_2014_to_bq"
+    - "inpatient_2015_transform_csv >> load_inpatient_2015_to_bq"
```

Comment:
For future reference, we can refactor this into a single KPod operator with 5x the resources, so we only spin up one pod and this `pipeline.yaml` becomes shorter and simpler too. No need to do this for now.